client/apollo/css/src/Form/ItemFile/ItemFileCommon.scss

```
@@ -0,0 +1,62 @@
+.af-item-file {
+  display: flex;
+  width: 30%;
+  margin-bottom: calc(10 / var(--font-size-base) * 1rem);
+  padding: calc(16 / var(--font-size-base) * 1rem);
+  border: 1px solid var(--item-file-border-color);
+  border-radius: var(--item-file-border-radius);
+  align-items: center;
+  justify-content: space-between;
+  gap: calc(12 / var(--font-size-base) * 1rem);
+
+  .af-item-file-title {
+    font: var(--font-family-base);
+    font-size: calc(18 / var(--font-size-base) * 1rem);
+    font-weight: 400;
+    line-height: 125%;
+    color: var(--item-file-title);
```

j'ai l'impression que c'est `--item-file-title-color` qu'il fallait mettre là
```suggestion
    color: var(--item-file-title-color);
```